draft-xyz-rats-psa-endorsements.md

```diff
@@ -61,28 +61,18 @@ informative:
 --- abstract
 
 PSA Endorsements comprise reference values, cryptographic key material and
-certification status information that a Verifier needs in order to appraise PSA
-Evidence.
-
-This memo describes the data formats used to convey PSA Endorsements to an Attestation Verifier.
-The format presented in this document is a PSA specific profile of CoRIM/CoMID based information model presented in
-Trusted Computing Group (TCG) Endorsement Architecture.
+certification status information that a Verifier needs in order to appraise
+attestation Evidence produced by a PSA device.  This memo defines such PSA
+Endorsements as a profile of the CoRIM data model.
```

Comment:
Do we want to refer or clarify more of the CoRIM data model? Where it exists or a link to its reference ?

Reply:
L75 has a TBD to be filled as soon as CoRIM gets a reference-able publication